tableaudocumentapi/field.py

```diff
@@ -24,6 +24,21 @@
 ]
 
 
+def argument_is_one_of(*allowed_values):
```

Comment:
It's TSC ;)
I've created a new module and put the decorator functions in there.